iis-wf/iis-wf-ingest-pmc/src/test/resources/eu/dnetlib/iis/wf/ingest/pmc/metadata/sampledataproducer/data/fault.properties

```diff
@@ -0,0 +1,3 @@
+inputObjectId=50|od_______908::14ddacb589be0a68489f8981_invalid
+code=org.jdom.input.JDOMParseException
+message=Error on line 1: Content is not allowed in prolog.
```

Comment:
I'm not sure if we the name (extension) of the file is the best one. No real properties are in this file, but rather expected faults.